testdata/networking/networkpolicy/multitenant_policy.yaml

```diff
@@ -0,0 +1,55 @@
+---
+
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: deny-by-default
+spec:
+  podSelector:
+  ingress: []
+
+--- 
+
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-openshift-ingress
```

Comment:
Just curious, if networkpolicy/ingress not allow ingress and monitoring, what will happen in the cluster?

Reply:
if that. route will cannot be accessed and all pods 
openshift-monitoring cannot be accessed the test pods, means prometheus cannot collect the data of this test pods

Reply:
/lgtm

Reply:
if that. route will cannot be accessed and all pods 
openshift-monitoring cannot be accessed the test pods, means prometheus cannot collect the data of this test pods